vsphere/datadog_checks/vsphere/event.py

```diff
@@ -148,14 +158,17 @@ def get_agg_key(alarm_event):
                 md5(alarm_event.alarm.name.encode('utf-8')).hexdigest()[:10],
             )
 
-        # Get the entity type/name
-        if self.raw_event.entity.entity.__class__ == vim.VirtualMachine:
-            host_type = 'VM'
-        elif self.raw_event.entity.entity.__class__ == vim.HostSystem:
-            host_type = 'host'
```

Comment:
Any reason why we removed these lines? I don't see how the `host_type` will be set to `host` with the new changes.

Reply:
Yes! This logic is done here https://github.com/DataDog/integrations-core/pull/15992/files#diff-c84b7cfa9c0e39f7412061ef4ac643a1b07c274435f7978a430ba6c057eb9354R69, we only need to do it for VM since it is capitalized rather than lowercase

Reply:
🤦 I was confused why you didn't use `MOR_TYPE_AS_STRING` because I thought that the `host_type` was initialized to `None` here https://github.com/DataDog/integrations-core/pull/15992/files#diff-c84b7cfa9c0e39f7412061ef4ac643a1b07c274435f7978a430ba6c057eb9354R161 so I looked under that line. (it was the `host_name`)